consensus/consensus_v2.go

```diff
@@ -43,6 +43,13 @@ func (consensus *Consensus) HandleMessageUpdate(ctx context.Context, msg *msg_pb
 		return nil
 	}
 
+	// when node is not in ViewChanging mode, ignore viewchange / newview messages
+	if !consensus.IsViewChangingMode() &&
```

Comment:
there are already logic to filter messages in validateShardBoundMessage. 
```	// when node is in ViewChanging mode, it still accepts normal messages into FBFTLog
	// in order to avoid possible trap forever but drop PREPARE and COMMIT
	// which are message types specifically for a node acting as leader
	// so we just ignore those messages
	if node.Consensus.IsViewChangingMode() {
		switch m.Type {
		case msg_pb.MessageType_PREPARE, msg_pb.MessageType_COMMIT:
			return nil, nil, true, nil
		}
	} else {
		// ignore newview message if the node is not in viewchanging mode
		switch m.Type {
		case msg_pb.MessageType_NEWVIEW:
			return nil, nil, true, nil
		}
	}```
No need to add filter here. (the new logic is different, which we shouldn't change)

Reply:
good point. let me check. I'll ignore viewchange message in the validation process.